apps/frontend/src/components/global/Sidebar.vue

```diff
@@ -124,11 +132,31 @@ import {views} from '@/store/app_info';
     ChecklistTargetDataModal,
     ChecklistTechnologyAreaModal,
     QuickFilters,
-    CategoryFilters,
+    DropdownFilters,
     SelectedFilterTable
   }
 })
 export default class Sidebar extends mixins(RouteMixin) {
+  /** Category list for in-file filtering dropdown */
+  readonly inFileFilterProps = [
```

Comment:
i guess this is fine for right now, but it would be nice if as part of the effort to prep for page specific filtering/views to move this content to the store so that it can be dynamically retrieved depending on what page we're viewing (ex. some of these are applicable to checklists but not results)

to make it symmetrical, maybe the file metadata one should be moved too, esp if/when we add support for more results specific things like an underlying profile/source filter (ex. filter to only inspec/redhat 7 profiles or to only asff origin findings)